packages/connect-iframe/webpack/utils.ts

```diff
@@ -0,0 +1,37 @@
+import path from 'path';
+
+type Project = 'iframe' | 'suite-web';
+
+export const getDistPathForProject = (project: Project = 'iframe') => {
+    const basePath = path.join(__dirname, '..', '..');
+    switch (project) {
+        case 'iframe':
+            return path.join(basePath, 'connect-iframe', 'build');
+        case 'suite-web':
+            return path.join(basePath, 'suite-web', 'build', 'static', 'connect');
```

Comment:
Yes, this is for using the same webpack configs for both projects.
So it is built directly from `suite-build` instead of copying the build from `connect-iframe` as was done before.